src/app/landing-page/landing-page.component.scss

```diff
@@ -115,3 +115,28 @@
 @import "assets/blk/scss/blk-design-system/angular/alerts";
 @import "assets/blk/scss/blk-design-system/angular/datepicker";
 @import "assets/blk/scss/blk-design-system/angular/example-pages";
+
+footer.footer{
+  background: #f9cf00 !important;
+  border-top-color: #f9cf00;
+  .btn-social i{
+    font-size: 30px !important; 
+    display: flex !important;
+    justify-content: center !important;
+    align-items: center;
+  }
+  
+  .btn-social a{
+    margin-left: 5px;
+    margin-right: 5px;
+    padding: 25px !important;
+  }
+
+  h3, p, i{
+    color: black !important;
+  }
+  
+  a{
+    border-color: black !important;
+  }
+}
```

Comment:
Al final de los archivos siempre deja una línea en blanco. La razón es [esta](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).